src/bench/load_external.cpp

```diff
@@ -46,7 +46,7 @@ static void LoadExternalBlockFile(benchmark::Bench& bench)
     ss << static_cast<uint32_t>(benchmark::data::block413567.size());
     // We can't use the streaming serialization (ss << benchmark::data::block413567)
     // because that first writes a compact size.
-    ss << Span{benchmark::data::block413567};
+    ss << std::span{benchmark::data::block413567};
```

Comment:
Previously
```C++
ss << Span{benchmark::data::block413567};
```
called:
```C++
template <typename Stream, BasicByte B> void Serialize(Stream& s, Span<B> span) { s.write(AsBytes(span)); }
```

But after the change it calls:
```C++
template <typename Stream, BasicByte B, std::size_t N> void Serialize(Stream& s, std::span<B, N> span) { s.write(std::as_bytes(span)); } //
```

The result in this case seems to be the same, so we might as well remove the extra span around the spans now:
```suggestion
    ss << benchmark::data::block413567;
```

Reply:
> Previously, the data was contained in a vector

A span is generated now:

```C++
inline constexpr std::span block413567{detail_block413567_raw};
```
So I don't see the reason for wrapping it in another span now.